src/styles/blocks/catalog.scss

```
@@ -17,6 +17,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    
+    transition: transform 300ms ease-in-out,
+      color 300ms ease-in-out ;
```

Create variables for this & replace them in all cases
```suggestion
    transition: transform $transitionDuration ease-in-out, color $transitionDuration ease-in-out;
```